tutorials/config.md

```diff
@@ -43,3 +46,78 @@ The priority of arguments is:
 3.  `ACTIONHERO_CONFIG=PATH1,PATH2 npm start`
 
 Note that if `--config` or `ACTIONHERO_CONFIG` are used, they _overwrite_ the use of the default `/config` folder. If you wish to use both, you need to re-specify "config", e.g. `--config=config,local-config`. Also, note that specifying multiple `--config` options on the command line does exactly the same thing as using one parameter with comma separators, however the environment variable method only supports the comma-delimited syntax.
+
+
+## Exemple
+
+
+- Configure your database with your key word ```db``` 
```

Comment:
I don't think I follow...  For a file `config/db.ts` you would export the config namespace `db` which would be used by `config.db.stuff` in the rest of the application